swagger.yaml

```diff
@@ -61,9 +63,86 @@ paths:
           description: Wrong content type (should be JSON)
         422:
           description: Missing fields
+  
+  /sessions/{sessionId}:
```

Comment:
I'm not really convinced that this is a right way to send secrets. I prefer to use e.g. headers here, because it is potentially safer (someone can add https between containers) and won't log session id by accident.

Reply:
I don't really see the argument for logging it and I wrote the endpoint with the idea that it would not be visible to the public.

Reply:
I could make the endpoint authorized using sessionId instead I guess.